style.css

```diff
@@ -0,0 +1,62 @@
+@import url('https://fonts.googleapis.com/css2?family=Poppins:ital,wght@0,100;0,200;0,300;0,400;0,500;0,600;0,700;1,100;1,200;1,300;1,500;1,600;1,700&display=swap');
+
+html {
+  height: 718vh;
```

Comment:
You can't set a defined height for a web app. Please remove it. Doing this adds a lot of horizontal scroll to your web app even though you have content that fits on the screen vertically. This is not a good practice.